### PR TITLE
restructure: move all lib js files into js/lib/*

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

Moving all imported js lib files into it's own folder in an attempt to make our folder structure a lil neater

### CODE REVIEW

**Comment by danlipert** [comment]
Whats the thinking behind moving some of the files to `/lib/` while leaving others alone? I do agree our js directories could use some cleaning up for sure 👍

**Comment by thelostone-mc** [comment]
@danlipert Ah so all those are files which aren't written by us -> aka library files 
Figured I'd start by taking the step to moving all of them to it's own folder so that's it's evident what we are using at the moment from outside.

Step 2: would be to restructure js files to make them more organized

File: app/kudos/templates/kudos_details.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     {% include 'shared/head.html' %}
     {% include 'shared/cards_pic.html' %}
     <link rel="stylesheet" href="{% static "v2/css/kudos/styles.css" %}" />
-    <script src="{% static 'v2/js/tweenlite.js' %}"></script>
+    <script src="{% static 'v2/js/lib/tweenlite.js' %}"></script>
```
Comment (octavioamu) [inline app/kudos/templates/kudos_details.html]: can we move this to the footer?

**Comment by octavioamu** [comment]
Left a comment but LGTM

**Comment by octavioamu** [comment]
Nice move, this will be very good also if we decide to jump into a bundle approach, like concat + minify js files having libs.js and  scripts.js called and not all separated files.

**Comment by thelostone-mc** [comment]
@octavioamu addressed comments + resolved conflicts

**Pull request merged**